### PR TITLE
Updating Travis CI definitions.

### PR DESCRIPTION
**Pull request by glauco**
This change updates .travis.yml to run our tests and
deploy to Github Pages automatically.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,10 +1,9 @@
-sudo: false
-
 language: ruby
-
 rvm:
-  - 2.0.0
```
Comment (bltavares) [inline .travis.yml]: :+1:

**Comment by bltavares** [comment]
Given that we are only depending on a single ruby version, would it make sense to remove `rvm` at all?

**Comment by bltavares** [comment]
Nevermind, they will fallback to 1.9.3 if it is not specified.

**Comment by bltavares** [comment]
:ship: :it:

**Comment by bltavares** [comment]
Does it need a filter on the `after_success:` to not run it on pull requests?

**Comment by bltavares** [comment]
Do we need the public key on the source control?

**Comment by glauco** [comment]
About the public key: I believe they are called public for a reason! :)

I see no harm in letting it to the repository.

**Comment by bltavares** [comment]
LGTM

**Pull request merged**